modules/acquia_cms_tour/src/Controller/DashboardController.php

```diff
@@ -147,10 +140,19 @@ public function content() {
     $form['help_text'] = [
       '#type' => 'markup',
       '#markup' => $this->t("ACMS organizes its features into individual components called modules.
-       The configuration dashboard/wizard setup will help you setup the pre-requisites.
-       Please note, not all modules in ACMS are required by default, and some optional modules
-       are left disabled on install. A checklist is provided to help you keep track of the tasks
-       needed to complete configuration."),
+      The configuration dashboard/wizard setup will help you setup the pre-requisites.
+      Please note, not all modules in ACMS are required by default, and some optional modules
+      are left disabled on install. A checklist is provided to help you keep track of the tasks
+      needed to complete configuration."),
+    ];
+    $form['empty_message'] = [
+      '#type' => 'markup',
+      '#markup' => $this->t("ACMS organizes its features into individual components called modules.
```

Comment:
@panshulK Can we italicized it ? Also, we should re-write below. Because there could be other plugins as well which depends on other modules.
> Please enable any of the following modules to be able to access the forms
      (ex. google_analytics, google_tag, gecoder, captcha, acquia_telemetry, cohesion)."),

Reply:
Done!